scripts/households.py

```diff
@@ -0,0 +1,39 @@
+
+
+import numpy as np
+import elliptical_u_est as ellip
+
+# household functions
+
+def HHFOCs(b_sp1, *params):
```

Comment:
I think it's better to rename this to FOCs to be consistent with other scripts where `households.py` is used (e.g. SS)